test/project_types/script/layers/infrastructure/languages/assemblyscript_task_runner_test.rb

```diff
@@ -274,4 +289,15 @@
       end
     end
   end
+
+  describe ".check_system_dependencies!" do
+    subject { as_task_runner.check_system_dependencies! }
+
+    it "should call the methods" do
+      as_task_runner.expects(:check_tool_version!).with("npm", "5.2.0")
+      as_task_runner.expects(:check_tool_version!).with("node", "14.5.0")
```

Comment:
I think we can safely ignore this for now since Thenura's other PR removes that method https://github.com/Shopify/shopify-cli/pull/1634/files#diff-b9b0b5a24b8075e43ce13a14411028b6fcb8fc0640e855610dbb00f02164e95cL31-L35 🙈 

However to bike shed, I think this should still be `.expects`. There is a chance that `check_tool_version!` raises an error and ends the command execution, which is a side effect. This method depends on that side effect as we should not continue execution if the user does not have the proper tool version installed